text/XXX-A-Compilation-of-Short-Interactive-Tutorials-and-How-To-Guides-for-Coq.md

```diff
@@ -0,0 +1,257 @@
+- Title: A Compilation of Short Interactive Tutorials and How-To Guides for Coq
```

Comment:
You may want to change "XXX" in the filename to the CEP number.  I believe that is a convention.

Reply:
In the PR they are many conventions used about that